src/lib/components/Creatibutors/CreatibutorsModal.js

```diff
@@ -196,11 +197,48 @@ export class CreatibutorsModal extends Component {
     }
   };
 
+
+  makeIdEntry = (identifier) => {
+
+  let icon = null;
+  let link = null;
+  let linkTitle = null;
+
+  if (identifier.scheme == "orcid") {
+    icon = "/static/images/orcid.svg";
+    link = "https://orcid.org/" + identifier.identifier;
+  } else if (identifier.scheme == "gnd") {
+    icon = "/static/images/gnd-icon.svg";
+    link = "https://orcid.org/" + identifier.identifier;
```

Comment:
Yes, I think it's a necessary evil because of the complexity regarding consuming and then first having to clean up before sending back metadata to the REST API.

Shelved: I will open an issue for this with some examples around the code, so we can see if it can be addressed.